### PR TITLE
Chore: Use illustrations from `artwork` repository

### LINKED ISSUE #518: Use illustrations from `artwork` repository
Let's not duplicate things and use the illustrations from https://github.com/webhintio/artwork (published as [`@hint/artwork`](https://www.npmjs.com/package/@hint/artwork) on `npm`).

### PR DESCRIPTION
**Pull request by RoryDuncan**
Fix #518 

# Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/webhint.io)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

## Short description of the change(s)

- Added postinstall.js
- Added postinstall script to package.json
- Added `@hint/artwork` dependency
- Removed files from repo that would be copied from `@hint/artwork` during postinstall
- Added .gitignore for files that would be copied from `@hint/artwork` during postinstall

### CODE REVIEW

**Comment by RoryDuncan** [comment]
Looks like I have some code style changes to make for the build.

File: package.json [hunk 1]
```diff
@@ -15,6 +15,7 @@
   },
   "devDependencies": {
     "@alrra/travis-scripts": "^3.0.1",
+    "@hint/artwork": "^2.2.0",
```
Comment (sarvaje) [inline package.json]: Is it `devDependency` ok?

Comment (alrra) [reply package.json]: Yes, we are copying them on `postinstall`, so during development. We have them "locally" after that.

**Comment by alrra** [comment]
@RoryDuncan Thank you!

**Comment by alrra** [comment]
@RoryDuncan [Changes are now live](https://webhint.io/), thanks again!

**Pull request closed without merging**